src/clic.adoc

```diff
@@ -698,10 +697,11 @@ The global interrupt-enable bit of the handler's privilege mode is
 cleared, to prevent preemption by higher-level interrupts in the same
 privilege mode.
 
-The overall behavior is summarized in the following table: the Current
+The overall behavior of interrupts with `clicintip[__i__]` and `clicintie[__i__]` asserted 
+is summarized in the following table: the Current
 `p/ie/il` fields represent the current privilege mode `P` (not
 software visible), interrupt enable `ie` =
-({status}.{ie} & `clicintie[__i__]`)  and interrupt
+{status}.{ie} and interrupt

```

Comment:
it is just a hinch, but maybe we are missing something now with delegation and virtual interrupts in AIA.
One reason why I am in favor of specifying differential rather than overall behavior.

Reply:
seems fine. i checked this table from AIA:
<img width="362" alt="image" src="https://github.com/user-attachments/assets/439b86f5-0573-44db-827f-b1f2fc44c9cd" />
and this is not in conflict.